lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -13,7 +13,7 @@ class Base
           ::RuntimeError,
         ].freeze
 
-        BASE_POINTS = 1_500_000
+        BASE_POINTS = 300_000 # 30 minutes (5min = 50K)
```

Comment:
Since you decided to include some commentary, WDYT of making it executable, via something like: `POINTS_PER_MINUTE = 10_000; BASE_POINTS = 30 * POINTS_PER_MINUTE`?

For now we'd end up with the same 10K constant in a few code-bases but maybe they get consolidated soon.